webpack/assets/javascripts/react_app/components/common/forms/ForemanForm/ForemanForm.stories.js

```diff
@@ -70,7 +70,7 @@ FormComponent.propTypes = {
   onCancel: PropTypes.func.isRequired,
 };
 
-storiesOf('Components/Form', module).add('Basic Foreman Form', () => (
+storiesOf('Components|Form', module).add('Basic Foreman Form', () => (
```

Comment:
an error occurs under Form section
![FireShot Capture 201 - Components - Form - FormField ⋅ Storybook - localhost](https://user-images.githubusercontent.com/11807069/71783553-ffe3d800-2ff0-11ea-955c-930e4d299411.png)

Reply:
This error already exists in the current storybook, see: https://github.com/theforeman/foreman/pull/7263